### ISSUE TITLE
mklocal.py fails withe permission issue

### ISSUE THREAD

**Issue by desepticon** [MilhouseVH/texturecache.py#55]
I'm having an issue with the mklocal.py script. It worked on my tvshows, but fails on my movies.

user@computer ~/s/m/texturecache>./texturecache.py jd movies | ./tools/mklocal.py --local / --prefix / --artwork fanart poster --output ./fixup.dat

Traceback (most recent call last):
  File "./tools/mklocal.py", line 872, in <module>
    main(init())
  File "./tools/mklocal.py", line 855, in main
    workitem = processItem(args, mediatype, media, download_items)
  File "./tools/mklocal.py", line 417, in processItem
    newname = processArtwork(args, mediatype, media, mediatitle, artitem["type"], mediafile, oldname, artpath_m, artpath_s, is_excess=False)
  File "./tools/mklocal.py", line 498, in processArtwork
    fname = pathToXBMC(getImage(args, mediatype, media, title, atype, filename, currentname, target))
  File "./tools/mklocal.py", line 543, in getImage
    shutil.copyfile(newsource, target)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py", line 97, in copyfile
    with open(dst, 'wb') as fdst:
IOError: [Errno 13] Permission denied: '/Volumes/w-fanart.jpg'

There is no file at /Volumes/w-fanart.jpg. I set all the permissions to 777 on my media drives just to make sure it wasn't a permissions issue.

Running MacOS 10.13.6

**Comment by MilhouseVH** [comment]
One of your movies most likely has that path.

Try running
```
texturecache.py jd movies >/tmp/movies.txt
```
and searching `/tmp/movies.txt` for `w-fanart.jpg` (and fixing the corresponding movie).

Or run
```
texturecache.py @logfile=/tmp/tc.log jd movies >/dev/null
```
then upload `/tmp/tc.log` somewhere (dropbox, googledrive etc.) and post the download link for the file.

**Comment by desepticon** [comment]
Ok. I was able to get passed the error by running as su. And I did end up with those two files at /Volumes.  But it seems strange. The errored files are not located at that path. They are located at /Volumes/wd.4tb.1/Movies/Jungle.2.Jungle.1997.DVD-Rip/

   331411 bytes -   fanart   - Jungle 2 Jungle                               -> /Volumes/w-fanart.jpg
   600736 bytes -   poster   - Jungle 2 Jungle                               -> /Volumes/w-poster.jpg
